BookPlayer/Player/PlayerManager.swift

```diff
@@ -772,6 +774,19 @@ extension PlayerManager {
       self.audioPlayer.seek(to: CMTime(seconds: newPlayerTime, preferredTimescale: CMTimeScale(NSEC_PER_SEC)))
     }
   }
+  
+  func handleSticky(_ item: PlayableItem) {
+    
+    let interval = SleepTimer.shared.lastInteral
+    if(SleepTimer.shared.sticky == true){
+    
+      if SleepTimer.shared.state == .off {
+        SleepTimer.shared.setTimer(.countdown(interval))
+      } else {
+        SleepTimer.shared.restartTimer()
+      }
+    }
+  }
```

Comment:
we would only need to check if the sticky preference is On, and in that case call `restartTimer`, and it should automatically set the last timer that was active

Reply:
Hmm it seems to me like that doesn't really work, seems to not restart the timer when getSticky return true 🤔

```
func handleStickyTimer() {
    
    if(SleepTimer.shared.getSticky()){
      SleepTimer.shared.restartTimer()
    }
  }
  ```

Reply:
hmmmm 🤔 I guess between app launches it wouldn't work, but it should work after setting the first timer? (let me know if it still doesn't work) if that's the case we could store the last active timer so on init we set that value on `lastActivedState` 🤔

Reply:
ah ok, it seems like playback is actually crashing in my simulator for whatever reason. So I suppose I can't test if it works, but I think it should.